src/shared/utilities/vsCodeUtils.ts

```diff
@@ -0,0 +1,169 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+import * as vscode from 'vscode'
+import * as nls from 'vscode-nls'
+import { EnvironmentVariables } from '../environmentVariables'
+export const localize = nls.loadMessageBundle()
+
+// TODO: Why are messages from bundle never displayed?
```

Comment:
Can you elaborate on this? Not sure I understand.

Reply:
Yes, if there is a massage for a key in the NLS file and a default message provided in the call, the one in the call seems to always be displayed. I thought it should display the value from the NLS file and fall back to the value provided at function call time if nothing matching that key is provided in the NLS file. Is it broken?

Reply:
@micholas I think that's because we don't yet provide any translations. So the fallback will always be used. But I'm a little surprised that it affects English.

Reply:
Ya, seems broken but no need to worry about it too much for now.